### PR TITLE
Create girijasgodbole.md

### PR DESCRIPTION
**Pull request by girijasgodbole**
https://rawgit.com/girijasgodbole/girijasgodbole.github.io/add-girija-profile/#!pages/vi/profiles/girijasgodbole.md

### CODE REVIEW

**Comment by sahilph** [comment]
@girijasgodbole 
Please use markdown instead of html for image display. Also if you could increase the content(add some more info) and use other markdown elements (like lists), it would be much better.

And please fix the rawit link in your PR description.

File: pages/vi/profiles/girijasgodbole.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Hey There!!
+* * *
+My name is **Girja Godbole**. 
+My current location is Atlanta, GA, USA (Eastern Standard Time).
+I am using Mac OS Sierra Version 10.12.4
+* * *
+
+### ABOUT ME <img src="http://2.bp.blogspot.com/-8P_iI3YueO0/T73feHn5VSI/AAAAAAAAAiw/N5N3HmVmk9I/s1600/6a00d834516a0869e2016760f339c3970b-800wi.jpg" width="64"/>
```
Comment (sahilph) [inline pages/vi/profiles/girijasgodbole.md]: Please use markdown instead of html for image display.

**Comment by girijasgodbole** [comment]
Thank you. Edited @sahilph

**Comment by sahilph** [comment]
@girijasgodbole Approved. 
For faster response, please send a message (with PR link) on gitter if you make any changes to the PR or an issue.

**Pull request merged**